DFC.App.JobProfile.CurrentOpportunities.Data/Models/CurrentOpportunitiesSegmentData.cs

```diff
@@ -0,0 +1,20 @@
+﻿using DFC.App.JobProfile.CurrentOpportunities.Data.Contracts;
+using System.Collections.Generic;
+
+namespace DFC.App.JobProfile.CurrentOpportunities.Data.Models
+{
+    public class CurrentOpportunitiesSegmentData : IDataModel
```

Comment:
Can you rename to CurrentOpportunitiesSegmentDataModel

Reply:
Done